XPlat/Google.Grpc/source/Xamarin.Grpc.Android.Bindings.XamarinAndroid/Xamarin.Grpc.Android.Bindings.XamarinAndroid.csproj

```diff
@@ -0,0 +1,100 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!--
+  https://montemagno.com/converting-xamarin-libraries-to-sdk-style-multi-targeted-projects/
+    
+  -->
+    <PropertyGroup>
+        <AssemblyName>Xamarin.Grpc.Android</AssemblyName>
```

Comment:
Because this project already has the Android in the name, we might as well leave off the `.Bindings.XamarinAndroid` since this will ALWAYS be a binding and Android is in the name. Same with the rest.

This project can just be named `Xamarin.Grpc.Android`. It will also be quite a few less characters.

Reply:
If you check you will see that pattern was:

for namespaces:
```
Xamarin.Grpc.Core
Xamarin.Grpc.Context
Xamarin.Grpc.Stub
Xamarin.Grpc.OkHttp
Xamarin.Grpc.Protobuf.Lite
```
and for projectnames - add sufix `Bindings.XamarinAndroid`

Why `Bindings`? I because I have templates where I have non-bindings projects for pure Xamarin.Android platform specific code. (I know that can go to `./Additions/`, but it was template decision choice). And if I need it I simply delete project generated by template.

Why `XamarinAndroid`? To avoid issues if namespace is `Android`.

And now `Android`. Google added new Grps artifact `grpc-android`.

https://mvnrepository.com/artifact/io.grpc/grpc-android

yes. I'm aware this might force users into using FQ typenames with prefix `global::`, but it was google that added artifact with the name that is not ideal for us.

Reply:
Please, leave this less important style issue as is. Let us agree on naming and I'll obey. Check Jon's reviews week or two ago - this was not the issue.

New artifact with unlucky name came in, but changing namespace names would be breaking change.

Changing projectnames is low priority for me. Unless Hugo.Boss has different opinion.